test/spec/browser/request-strategy/only-JSONP-when-XHR-failure.js

```diff
@@ -1,63 +0,0 @@
-'use strict';
```

Comment:
Indeed, so a test that would first use XHR (fail), then JSONP and then use XHR again?